proposals/nnnn-screen-manager-scrollable-message-and-slider.md

```diff
@@ -0,0 +1,212 @@
+ */
+@property (copy, nonatomic, readonly) NSString *text;
+
+/**
+ Maps to ScrollableMessage.timeout. Defaults to `defaultTimeout`. If set to 0, it will use `defaultTimeout`. The minimum is 3 seconds; the maximum is 65 seconds. If this is set below the minimum, it will be capped at 1 second. If this is set above the maximum, it will be capped at 65 seconds. Defaults to 0.
+ */
+@property (assign, nonatomic, readonly) NSTimeInterval timeout;
+
+/**
+ Maps to Alert.softButtons. Soft buttons the user may select to perform actions. Only one `SDLSoftButtonState` per object is supported; if any soft button object contains multiple states, an exception will be thrown.
+ */
+@property (nullable, copy, nonatomic, readonly) NSArray<SDLSoftButtonObject *> *buttons;
+
+- (instancetype)initWithText:(NSString *)text timeout:(NSTimeInterval)timeout buttons:(NSArray<SDLSoftButtonObject *>)buttons;
+
+/**
+ Cancels the scrollable message. If the RPC has not yet been sent to Core, it will not be sent. If the message is already presented on Core, it will be immediately dismissed. Canceling an already presented message will only work if connected to Core versions 6.0+. On older versions of Core, it will not be dismissed.
+*/
+- (void)cancel;
+
+@end
+```
+
+```objc
+@interface SDLSliderView
+/**
+ Set this to change the default timeout for all sliders. If a timeout is not set on an individual view (or if it is set to 0.0), then it will use this timeout instead. See `timeout` for more details. If this is not set by you, it will default to 10 seconds. The minimum is 1 second, the maximum is 65 seconds. If this is set below the minimum, it will be capped at 1 second. If this is set above the maximum, it will be capped at 65 seconds.
+ */
+@property (class, assign, nonatomic) NSTimeInterval defaultTimeout;
+
+/**
+ Maps to Slider.timeout. Defaults to `defaultTimeout`. If set to 0, it will use `defaultTimeout`. The minimum is 3 seconds; the maximum is 65 seconds. If this is set below the minimum, it will be capped at 1 second. If this is set above the maximum, it will be capped at 65 seconds. Defaults to 0.
+ */
+@property (assign, nonatomic, readonly) NSTimeInterval timeout;
+
+/**
+ Maps to Slider.sliderHeader. The text that is displayed as a title describing the modal.
+*/
+@property (copy, nonatomic, readonly) NSString *title;
+
+/**
+ Maps to Slider.position. The initial location of the slider handle. Must be a value between 1 and 26 and cannot exceed `numPositions` or an exception will be thrown.
+*/
+@property (assign, nonatomic, readonly) NSUInteger initialPosition;
+
+/**
+ Maps to Slider.numTicks. The number of values that will be available to the user on the slider. If the slider is created as a 'static' slider, then there will be one "footer" value and the 'ticks' on the slider will not have individual values. Must be a number between 2 and 26 or an exception will be thrown.
```

Comment:
```suggestion
 Maps to Slider.numTicks. The number of values that will be available to the user on the slider. If the slider is created as a 'static' slider, then there will be one 'footer' value and the 'ticks' on the slider will not have individual values. Must be a number between 2 and 26 or an exception will be thrown.
```

Also did you intend to use the apostrophe instead of backtick here?  If not, please update!

Reply:
I did because it's not referring directly to a parameter name.